server/database.js

```diff
@@ -1,28 +1,90 @@
 const mysql = require('mysql');
+require('dotenv').config();
 
 const loginInfo = {
-    host: 'mycoolsql',
-    port: '3306',
-    user: 'root',
-    password: 'sheep',
-    database: 'featurefilm'
+  host: 'mycoolsql',
+  port: '3306',
+  user: 'root',
+  password: 'MattM',
+  database: 'featurefilm',
 };
 
 const connection = mysql.createConnection(loginInfo);
 
+// create
+const createFeaturedFilm = (params) => {
+  const createQuery = `INSERT INTO Features (
+    title,
+    category_1,
+    category_2,
+    release_date,
+    mpaa_rating,
+    length,
+    star_rating,
+    star_rating_count,
+    rt_rating,
+    description,
+    hd_rent,
+    sd_rent,
+    hs_cost,
+    sd_cost,
+    movie_shot_url,
+    movie_cover_url
+  ) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)`;
+  return new Promise((resolve, reject) => {
+    connection.query(createQuery, params, (err, results) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(results);
+      }
+    });
+  });
+};
+// read
+const getFeaturedFilmById = (id) => {
+  const readQuery = `SELECT * FROM Features WHERE id = ${id}`;
+  return new Promise((resolve, reject) => {
+    connection.query(readQuery, (err, results) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(results);
+      }
+    });
+  });
+};
 
-const getFeaturedFilmById = (id, callback) => {
-  connection.query(`SELECT * FROM Features WHERE id = ${id}`, (err, result) => {
-    if(err){
-      //console.log(err);
-    } else {
-      callback(err, result);
-    }
-
-
+// update
+const updateFeaturedFilm = (params) => {
+  const updateQuery = 'UPDATE Features SET ?? = ? Where id = ?;';
+  return new Promise((resolve, reject) => {
+    connection.query(updateQuery, params, (err, results) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(results);
+      }
+    });
+  });
+};
+// delete
+const removeFeaturedFilm = (id) => {
+  const removeQuery = `Delete from Features where id=${id};`;
```

Comment:
Capitalizing your query syntax would make this easier to read. i.e. ```DELETE FROM features WHERE id=${id}```

Other than that, looks great to me!